docs/README-migration.md

```diff
@@ -802,10 +802,9 @@ On Haiku OS, SDL no longer sets the current working directory to the executable'
 
 ```c
 {
-    char *path = SDL_GetBasePath();
+    const char *path = SDL_GetBasePath();
     if (path) {
-        chdir(path);
-        SDL_free(path);
+        chdir(path);  /* don't free this string, SDL owns it. */
```

Comment:
I don't think you need the comment. We're not highlighting the string rule here, we're just showing how to get the SDL2 behavior.

Reply:
Fixed in bbffc3a4940cddefcc85a4fcd6312ef8011bd296.